### PR TITLE
First major Paramly commit.

### PR DESCRIPTION
**Pull request by ishaimika**
Started building Paramly.
Paramly is the common used backbone in PayFunc and CardFunc CLI..

### CODE REVIEW

File: .vscode/tasks.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        ]
+      },
+      {
+        "type": "npm",
+        "script": "test",
+        "problemMatcher": []
+      }
+    ]
+  }
+
```
Comment (simonmika) [inline .vscode/tasks.json:44]: Missing newline at end of file.

File: Application.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+	}
+	async run(argument: string[]): Promise<boolean> {
+		const result = await this.execute(argument.slice(2))
+		console.log(result ? "succeeded" : "failed")
+		return result
+	}
+	register(module: Module<T>, ...names: string[]): void {
+		names.forEach(name => this.modules[name] = module)
+	}
+}
```
Comment (simonmika) [inline Application.ts:85]: Missing newline at end of line.

File: .vscode/tasks.json (hunk 1 above)
Comment (ishaimika) [reply .vscode/tasks.json]: Fixed.

File: Application.ts (hunk 2 above)
Comment (ishaimika) [reply Application.ts]: Fixed.

**Pull request merged**